test/Org.test.js

```diff
@@ -34,79 +56,190 @@ describe("Org", function() {
+    assert.isDefined(org.address);
+
+    const activeClaim = await org.activeClaim();
+    assertEqualClaims(activeClaim, EmptyClaim, "Deployed with non-empty active claim");
+  });
+
   it("denies USER to construct", async function() {
     await expectRevert.unspecified(Org.new(123, this.endaomentAdmin.address, { from: user }));
   });
-  
+
   it("registers valid claim requests", async function() {
+    const claimId = uuidv4();
     const org = await Org.new(123, this.endaomentAdmin.address, { from: admin });
-    const claimRequestReceipt = await org.claimRequest("John", "Doe", "john@doe.com", user, { from: user });
-    const claimCount = await org.getClaimsCount({ from: user });
-    assert.equal(claimCount, 1);
-    const claim = await org.claims(0, {from: user});
-    expectEvent(claimRequestReceipt, "ClaimCreated", 
-        { claim: [
-            claim.firstName,
-            claim.lastName,
-            claim.eMail,
-            claim.desiredWallet,
-            claim.filesSubmitted,
-        ]}
-    );
+
+    const claimRequestReceipt = await org.claimRequest(claimId, "John", "Doe", "john@doe.com", user, { from: user });
+    expectEvent(claimRequestReceipt, "ClaimCreated",
+          {
+            claimId,
+            claim: [
+              "John",
+              "Doe",
+              "john@doe.com",
+              user,
+            ],
+          });
+
+    const claim = await org.pendingClaims(claimId);
+
     assert.equal(claim.firstName, "John");
     assert.equal(claim.lastName, "Doe");
     assert.equal(claim.eMail, "john@doe.com");
     assert.equal(claim.desiredWallet, user);
-    assert.equal(claim.filesSubmitted, true);
   });
-  
+
+  it("rejects a duplicate claimId", async function () {
+    const claimId = uuidv4();
+    const org = await Org.new(123, this.endaomentAdmin.address, { from: admin });
+
+    const claimRequest = await org.claimRequest(claimId, "John", "Doe", "john@doe.com", user, { from: user });
+    assert.isTrue(claimRequest.receipt.status, "Failed to deploy claim");
+
+    const secondClaimRequestTx = org.claimRequest(claimId, "John", "Doe", "john@doe.com", user, { from: user });
+
+    await expectRevert(
+      secondClaimRequestTx,
+      "Org: Pending Claim with Id already exists",
+    );
+  });
+
   it("allows ADMIN to approve claims", async function() {
     const org = await Org.new(123, this.endaomentAdmin.address, { from: admin });
-    const claimRequestReceipt = await org.claimRequest("John", "Doe", "john@doe.com", user, { from: user });
-    
-    const claim = await org.claims(0, {from: user});
-    const claimEventData = { claim: [
-        claim.firstName,
-        claim.lastName,
-        claim.eMail,
-        claim.desiredWallet,
-        claim.filesSubmitted
-    ]};
+    const claimId = uuidv4();
+    const claimRequestReceipt = await org.claimRequest(claimId, "John", "Doe", "john@doe.com", user, { from: user });
+
+    const claimEventData = { claimId,
```

Comment:
Style for this looks different than it does below

Reply:
Fixed!